src/modules/user/tests/user.controller.spec.ts

```diff
@@ -0,0 +1,64 @@
+// user.service.spec.ts
+import { Test, TestingModule } from '@nestjs/testing';
+import { UserService } from '../user.service';
+import { UserRepository } from '../user.repository';
+import { User } from '../user.entity';
+import { NotFoundException, UnauthorizedException } from '@nestjs/common';
+
+describe('UserService', () => {
```

Comment:
Please move this to a file named user.controller.e2e.ts, sine these are e2e tests.